src/Form/Procuration/ProcurationProxyType.php

```diff
@@ -46,6 +46,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('voteOffice', TextType::class)
             ->add('proxiesCount', ChoiceType::class, [
                 'expanded' => true,
+                'mapped' => false,
```

Comment:
why this change? should it not be mapped to `Proxy::proxiesCount`?

Reply:
nice catch